Sources/LogConsole/LogInterceptor.cs

```diff
@@ -10,6 +10,8 @@
 using System;
 using UnityEngine;
 
+using Logger = KSPDev.LogUtils.Logger;
```

Comment:
Formatting logging is a feature of Unity 5: https://docs.unity3d.com/ScriptReference/Debug.html

Don't waste your time on refactoring KIS code to these new methods. It's only nice to have. Eventually, all legacy code will go away anyways.